013-quarkus-oidc-restclient/src/test/java/io/quarkus/qe/OpenApiTest.java

```diff
@@ -0,0 +1,28 @@
+package io.quarkus.qe;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.junit.QuarkusTest;
+import io.smallrye.openapi.runtime.io.Format;
+
+@QuarkusTest
+public class OpenApiTest {
```

Comment:
Apart from the @rsvoboda 's comments, I'm missing the Native version of this test.